modules/capnproto/1.1.0/source.json

```diff
@@ -0,0 +1,9 @@
+{
+    "url": "https://github.com/capnproto/capnproto/archive/refs/tags/v1.1.0.tar.gz",
+    "integrity": "sha256-wKDXigfoIfe64mx/ysIKkgLrPWOaZzsmBrdgkqHzW2s=",
+    "strip_prefix": "capnproto-1.1.0",
```

Comment:
You can probably use `capnproto-1.1.0/c++` here so that you don't need all the patches? Looks like this is the Bazel root for this project anyway (where the WORKSPACE is located).

Reply:
I tried this out a couple of days ago and was not able to get it to work.
If I recall correctly I ran into patching issues.

Reply:
Thanks!